src/filerepository.js

```diff
@@ -171,7 +172,9 @@ export default class FileRepository extends Plugin {
 			 *
 			 * @error filerepository-no-upload-adapter
 			 */
-			log.error( 'filerepository-no-upload-adapter: Upload adapter is not defined.' );
+			console.error( attachLinkToDocumentation(
```

Comment:
`console.warn`? We use `console.warn` if there is no toolbar item. I believe this is a similar situation.